packages/autocomplete-tools/@types/fig/index.d.ts

```diff
@@ -3,25 +3,106 @@ export {};
 
 declare global {
   export namespace Fig {
-    // All the available templates
     type TemplateStrings = "filepaths" | "folders" | "history";
 
+    /**
+     * A template which is a single TemplateString or an array of TemplateStrings.
+     *
+     * @remark
```

Comment:
The correct syntax seems to be `@remarks`. We need the right one when using the parser provided by ts-docs. It affects the whole file, not just this one.